requirements.md

```diff
@@ -91,3 +91,37 @@
   User pastes unformatted text into the search bar
   1. User pastes unformatted text into the search bar
   2. System automatically formats text to default text and font
+
+5. Use Case Name: User Profile
+- **Pre-condition:**  User has access to wifi and has an account already created.
+ 
+- **Trigger:**  User clicks profile icon
+ 
+- **Primary Sequence:**
+  1. User wants to view their profile
+  2. User clicks on user icon
+  3. System identifies their account
+  4. System navigates user to their personal user profile
+ 
+- **Primary Postconditions:** User is able to view their personal profile and interact with their page.
+ 
+- **Alternate Sequence:**
+  1. User has not created a profile yet when clicking profile icon
+  2. System prompts user with create a profile screen
+ 
+6. Use Case Name: Send Messages to Followers
+- **Pre-condition:**  User has access to wifi and has an account already created.
+ 
+- **Trigger:**  User clicks the button to create a public post
+ 
+- **Primary Sequence:**
+  1. User clicks create post icon
+  2. System redirects user to draft screen for your message
+  3. User is able to input a series of strings or upload an image
```

Comment:
Can you remove the upload image part because there is another use case specifically for posting image with message?